client/src/views/Orders.vue

```diff
@@ -1,31 +1,177 @@
 <template>
   <div>
-    <h1>Orders</h1>
-    <div v-for="order in orders" :key="order.id">
-      <router-link :to="{ path: `/admin/order/${order.id}` }">
-        {{ order.id }}
-      </router-link>
+    <PageHeader>Orders</PageHeader>
+      <div class="container mx-auto my-10 md:my-12 px-4 flex flex-col md:flex-row">
+
+      <div class="md:w-1/5 mb-4 md:mb-0">
+          <h2 class="mb-5 uppercase text-sm tracking-wide text-black">
+            Order Type
+          </h2>
+          <div class="my-4" :key="deliveryType" v-for="deliveryType in deliveryTypes">
+            <input
+              type="checkbox"
+              class="checkbox hidden"
+              :id="deliveryType"
+              :value="deliveryType"
+              v-model="selectedDeliveryTypes"
+            />
+            <label :for="deliveryType" class="relative">
+              <span class="ml-2 leading-none">{{deliveryType}}</span>
+            </label>
+          </div>
+          
+          <h2 class="mb-5 uppercase text-sm tracking-wide text-black">
+            Order Status
+          </h2>
+          <div class="my-4" :key="status" v-for="status in statuses">
+            <input
+              type="checkbox"
+              class="checkbox hidden"
+              :id="status"
+              :value="status"
+              v-model="selectedStatuses"
+            />
+            <label :for="status" class="relative">
+              <span class="ml-2 leading-none">{{formatOrderStatus(status)}}</span>
+            </label>
+          </div>
+        </div>
+
+      <table class="table-auto md:w-4/5">
+        <tr class="bg-grey-light pb-8">
+          <th>Order ID</th>
+          <th>Status</th>
+          <th>Type</th>
+          <th>Date</th>
+          <th>Customer</th>
+        </tr>
+
+        <tr v-for="order in filteredOrders" :key="order.id" class="hover:bg-grey-lighter mt-10 text-center">
+          <td>
+            <router-link :to="{ path: `/admin/order/${order.id}` }">
+              {{ order.id }}
+            </router-link>
+          </td>
+          <td>{{ formatOrderStatus(order.status) }}</td>
+          <td>{{ (order.shippingAddress == null ? "Store Pick-Up" : "Delivery") }}</td>
```

Comment:
This could be simplified
```suggestion
          <td>{{ (order.shippingAddress ? "Store Pick-Up" : "Delivery") }}</td>
```